src/components/spaces/chart-test-space.tsx

```diff
@@ -0,0 +1,42 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "../base";
+import { TwoUpDisplayComponent } from "../two-up-display";
+
+import { PopulationsComponent } from "./populations/populations-container";
+import { ChartTest } from "../charts/chart-test";
+
+interface IProps extends IBaseProps {}
+interface IState {}
+
+@inject("stores")
+@observer
+export class ChartTestSpaceComponent extends BaseComponent<IProps, IState> {
```

Comment:
Is this component ever used, or accessible?

Reply:
not any more - I'll remove it, thanks